### PR TITLE
Fix BLS sigs / BLS curve naming confusion

### PR DESCRIPTION
**Pull request by veorq**

### CODE REVIEW

File: EIPS/eip-2334.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 eip: 2334
-title: BLS12-381 Deterministic Account Hierarchy
+title: BLS-BLS12-381 Deterministic Account Hierarchy
```
Comment (CarlBeek) [inline EIPS/eip-2334.md:3]: I think this does more to confuse than to clarify. Assuming the first "BLS" is for Boneh-Lynn-Shacham, is it important to acknowledge here? The only relevant item here is the curve order.

File: EIPS/eip-2334.md [hunk 2]
```diff
@@ -20,11 +20,11 @@ A standard for allocating keys generated by [EIP-2333](./eip-2333.md) to a speci
 
 ## A note on purpose
 
-This specification is designed not only to be an Ethereum 2.0 standard, but one that is adopted by the wider community who have adopted the BLS12-381 signature standard. It is therefore important also to consider the needs of the wider industry along with those specific to Ethereum. As a part of these considerations, it is the intention of the author that this standard eventually migrate to a more neutral repository in the future.
+This specification is designed not only to be an Ethereum 2.0 standard, but one that is adopted by the wider community who have adopted Boneh–Lynn–Shacham (BLS) signatures on the [BLS12-381 curve](https://electriccoin.co/blog/new-snark-curve/). It is therefore important also to consider the needs of the wider industry along with those specific to Ethereum. As a part of these considerations, it is the intention of the author that this standard eventually migrate to a more neutral repository in the future.
```
Comment (CarlBeek) [inline EIPS/eip-2334.md:23]: Again, I'd prefer to push readers towards the CFRG draft, if anything.

**Review by CarlBeek — CHANGES_REQUESTED** [review]
Thanks @veorq

I agree it is important to be precise with which "BLS" is being referred to and I appreciate the effort to be clear with "CFRG draft" terminology, but I don't agree with all of your phrasing.

File: EIPS/eip-2334.md (hunk 1 above)
Comment (veorq) [reply EIPS/eip-2334.md:3]: Agree, I thought I had removed this change, will revert

File: EIPS/eip-2334.md (hunk 2 above)
Comment (veorq) [reply EIPS/eip-2334.md:23]: makes sense, it's the authoritative reference that Eth uses, will add it

**Comment by veorq** [comment]
Thanks @CarlBeek for the review, will edit as commented.

File: EIPS/eip-2333.md [hunk 3]
```diff
@@ -21,13 +21,13 @@ In addition to the above, this method of deriving keys provides an emergency bac
 
 ## A note on purpose
 
-This specification is designed not only to be an Ethereum 2.0 standard, but one that is adopted by the wider community who have adopted the BLS12-381 signature standard. It is therefore important also to consider the needs of the wider industry along with those specific to Ethereum. As a part of these considerations, it is the intention of the author that this standard eventually migrate to a more neutral repository in the future.
+This specification is designed not only to be an Ethereum 2.0 standard, but one that is adopted by the wider community who have adopted the BLS signatures over BLS12-381. It is therefore important also to consider the needs of the wider industry along with those specific to Ethereum. As a part of these considerations, it is the intention of the author that this standard eventually migrate to a more neutral repository in the future.
```
[the remark this answers is not in the document]
Comment (veorq) [reply EIPS/eip-2333.md]: Addressed in https://github.com/veorq/EIPs/commit/888d25ed9054bdc85a704151b3a5324815beefd0, with other tweaks

File: EIPS/eip-2334.md (hunk 2 above)
Comment (veorq) [reply EIPS/eip-2334.md]: Please see https://github.com/veorq/EIPs/commit/17242295ce1be1ad573ac7b972cf2d7b35d9e737

File: EIPS/eip-2334.md (hunk 1 above)
Comment (veorq) [reply EIPS/eip-2334.md]: Please see https://github.com/veorq/EIPs/commit/17242295ce1be1ad573ac7b972cf2d7b35d9e737

File: EIPS/eip-2333.md [hunk 4]
```diff
@@ -21,13 +21,13 @@ In addition to the above, this method of deriving keys provides an emergency bac
...
+The curve (BLS12-381) used for BLS signatures within Ethereum 2.0 (alongside many other projects) mandates a new key derivation scheme. The most commonly used scheme for key derivation within Ethereum 1.x is [BIP32](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki) (also known as HD derivation) which deems keys greater than the curve order invalid. Based on the order of the private key subgroup of BLS12-381 and the size of the entropy utilised, more than 54% of keys generated by BIP32 would be invalid. (secp256k1 keys derived by BIP32 are invalid with probability less than 1 in 2<sup>-127</sup>.)
```
Comment (axic) [inline EIPS/eip-2333.md:30]: I think after removing `new` the parenthesis are confusing:
```suggestion
The curve BLS12-381 used for BLS signatures within Ethereum 2.0 (alongside many other projects) mandates a new key derivation scheme. The most commonly used scheme for key derivation within Ethereum 1.x is [BIP32](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki) (also known as HD derivation) which deems keys greater than the curve order invalid. Based on the order of the private key subgroup of BLS12-381 and the size of the entropy utilised, more than 54% of keys generated by BIP32 would be invalid. (secp256k1 keys derived by BIP32 are invalid with probability less than 1 in 2<sup>-127</sup>.)
```

Comment (veorq) [reply EIPS/eip-2333.md]: Right, overlooked this, thanks for spotting it

**Review by CarlBeek — APPROVED** [review]

File: EIPS/eip-2333.md [hunk 5]
```diff
@@ -152,7 +152,7 @@ Every key generated via the key derivation process derives a child key via a set
 * `"BLS-SIG-KEYGEN-SALT-"` is an ASCII string comprising 20 octets.
 * `OS2IP` is as defined in [RFC3447](https://ietf.org/rfc/rfc3447.txt) (Big endian encoding)
 * `I2OSP` is as defined in [RFC3447](https://ietf.org/rfc/rfc3447.txt) (Big endian decoding)
-* `r` is the order of the BLS 12-381 curve defined in [the draft IETF BLS signature scheme standard](https://tools.ietf.org/html/draft-irtf-cfrg-bls-signature-04) `r=52435875175126190479447740508185965837690552500527637822603658699938581184513`
```
Comment (lilchris502) [inline EIPS/eip-2333.md:155]: xpub6CA73cCqXUHCVV2EUoV3aaCiVDBeg28VCPDjnLaWEnCps1Q4CFBaxBXNDBpYwwxWrvE88Bhp53hreqhmqXJr8KDzAEPBrVUTtCqkEW6X8e9

**Review by lilchris502 — COMMENTED** [review]
Okay

**Comment by MicahZoltu** [comment]
@alita-moore Any idea why the bot ran and commented on this PR *after* it was merged?

**Comment by alita-moore** [comment]
@MicahZoltu I regression test old PRs

**Pull request merged**